enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -765,13 +768,26 @@ func (c *podmanCommandContainer) pullImage(ctx context.Context, creds container.
 			creds.Password,
 		))
 	}
+
+	if *podmanPullLogLevel != "" {
+		podmanArgs = append(podmanArgs, fmt.Sprintf("--log-level=%s", *podmanPullLogLevel))
+	}
+
 	podmanArgs = append(podmanArgs, c.image)
 	// Use server context instead of ctx to make sure that "podman pull" is not killed when the context
 	// is cancelled. If "podman pull" is killed when copying a parent layer, it will result in
 	// corrupted storage.  More details see https://github.com/containers/storage/issues/1136.
 	pullCtx, cancel := context.WithTimeout(c.env.GetServerContext(), *pullTimeout)
 	defer cancel()
-	pullResult := runPodman(pullCtx, "pull", &container.Stdio{}, podmanArgs...)
+	stdio := &container.Stdio{}
+	if *podmanPullLogLevel != "" {
+		stdio = &container.Stdio{
+			Stdin:  nil,
```

Comment:
nit: you don't need to specify keys with nil values

Reply:
Ah, thanks, removed it